scipy/stats/tests/test_distributions.py

```diff
@@ -181,6 +181,39 @@ def test_vonmises_logpdf(x, kappa, expected_logpdf):
     assert_allclose(logpdf, expected_logpdf, rtol=1e-15)
 
 
+def test_vonmises_expect():
+    """
+    Test that the vonmises expectation values are
+    computed correctly.  This test checks that the
+    numeric integration estimates the correct normalization
+    (1) and mean angle (loc).  These expectations are
+    independent of the chosen 2pi interval.
+    """
+    # check that expectation values
+    assert np.isclose(
+            stats.vonmises.expect(lambda x: 1, args=(1,)),
+            1)
```

Comment:
In case you get to this before me, I planned to refactor these like
```suggestion
    res = stats.vonmises.expect(lambda x: 1, args=(1,))
    assert_allclose(res, 1)
```